src/components/widget/Lookup/RawLookup.js

```diff
@@ -566,6 +562,7 @@ RawLookup.propTypes = {
   dispatch: PropTypes.func.isRequired,
   onDropdownListToggle: PropTypes.func,
   isComposed: PropTypes.bool,
+  field: PropTypes.string,
 };
 
-export default connect(mapStateToProps)(RawLookup);
+export default connect()(RawLookup);
```

Comment:
@siemiatj can you explain pls why filter from state is not needed here anymore ?

Reply:
I forgot to remove connect as a whole.